api-utils/apiHooks.ts

```diff
@@ -69,4 +69,31 @@ export class ApiHooks {
   public static async deleteGoalById(request: APIRequestContext, goalId: string) {
     await request.delete(`/api/v2/goal/${goalId}`);
   }
+
+  public static async createNewDocInSpace(request: APIRequestContext, docName: string, spaceId: string) {
+    const teamId: string = process.env.BASE_TEAM_ID as string;
+    const apiKey: string = process.env.API_KEY as string;
+    const createDocEndpoint: string = `https://api.clickup.com/api/v3/workspaces/${teamId}/docs`;
+
+    const newDocBody = {
+      name: docName,
+      parent: {
+        id: spaceId,
+        type: 4,
+      },
+    };
```

Comment:
Można się zastanowić czy wszystkie tego typu dane brać z klasy GenerateData

Reply:
W sumie racja, chociaż akurat w tym przypadku żadna dana nie jest generowana - jeżeli doc ma być przypisany do danego space, to [➡️ dokumentacja api](https://clickup.com/api/clickupreference/operation/createDoc/#tag/Docs/operation/createDoc!path=parent&t=request) wymaga podania konkretnego `spaceId`, oraz `type` musi być ustawione na `4`.

Reply:
Zawsze można zmienić nazwę tej klasy, która sugerowałaby, że klasa zajmuje się obsługą tego typu obiektów zamiast generowaniem danych. Drobny temat, nie ma się co przejmować.